packages/neo-one-website/docs/4-contributing/1-codebase-overview.md

```diff
@@ -158,7 +158,7 @@ Creates the environment for compiling smart contracts, which is then used in the
 
 ### neo-one-smart-contract-lib
 
-Defines NEP5 tokens and ICOs.
+Defines NEP17 tokens and ICOs.
```

Comment:
Actually revert this since NEP5 will still be NEP5 for Neo2 docs, which will still be produced from the `docs` folder. I've made this change to the new Neo3 docs folder on my branch.